README.md

```diff
@@ -90,6 +90,50 @@ Persistence Query usage example to obtain a stream with all events tagged with "
 Migrations
 ----------
 
+### Migrations to 0.59
+
+0.59 introduces a completely different way to manage tags for events.
+ 
+Migration requires some downtime but the new mechanism removes the restriction of only 3 tags 
+and improves performance, scalability and reliability of eventsByTag queries.
+
+* Migrate data to new table
+
+FIXME: Implement and document
+
+* `pubsub-minimum-interval` has been removed. If using DistributedPubsub notifications for tag writes then set `pubsub-notification` to on
+
+* `delayed-event-timeout` has been replaces by `events-by-tag-gap-timeout` with the restriction removed that all
+events have to be tagged. 
+
+* `first-time-bucket` format has changed to: `yyyyMMddTHH:mm` e.g. `20151120T00:00`
```

Comment:
we could still support the old format, which is still ok for day buckets

Reply:
:+1:  added support for this